From de6099679f6a0c7e4363ce2aa34c0ddd832d799a Mon Sep 17 00:00:00 2001
From: christophecvr <stefansat@telenet.be>
Date: Sun, 20 Nov 2016 14:12:16 +0100
Subject: [PATCH] Gst-plugins-good compile error with gcc > 6.x

 With last master gst-plugins-good we do have a compile error:
 when using gcc > 6.x and [-Werror=maybe-uninitialized] on.
 This patch solves the compile error so users can keep this
 quite important -Werror on by the compiler.

	modified:   gst/isomp4/qtdemux.c
---
 gst/isomp4/qtdemux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/isomp4/qtdemux.c b/gst/isomp4/qtdemux.c
index 15ac144..b76e103 100644
--- a/gst/isomp4/qtdemux.c
+++ b/gst/isomp4/qtdemux.c
@@ -10194,7 +10194,7 @@ qtdemux_parse_trak (GstQTDemux * qtdemux, GNode * trak)
                * to find one called field. Walk through them. */
               off_t offset = min_size;
               while (offset + 8 < len) {
-                guint32 size, tag;
+                guint32 size = 0, tag;
                 ok = gst_byte_reader_get_uint32_le (&br, &size);
                 ok &= gst_byte_reader_get_uint32_le (&br, &tag);
                 if (!ok || size < 8) {
-- 
2.7.4

